kubernetes/assets/service_checks.json

```diff
@@ -5,15 +5,15 @@
         "check": "kubernetes_state.node.ready",
         "statuses": [
             "ok",
-            "unknown",
+            "warning",
```

Comment:
to keep compatiblity with `kubernetes_state` (python check)
I think we need to keep `unknown` too

Reply:
The python check also uses warning instead of unknown.

Here you can see that it uses “condition_to_status_positive”: https://github.com/DataDog/integrations-core/blob/0b2db55128cfacc7406cff9c18027a33e7d1f0a7/kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py#L793 which means that it only uses "ok", "critical", and "warning": https://github.com/DataDog/integrations-core/blob/0b2db55128cfacc7406cff9c18027a33e7d1f0a7/kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py#L96